mkdocs.yml

```
@@ -0,0 +1,68 @@
+    - navigation.indexes
+    - navigation.expand
+  font:
+    text: Rubik
+    code: Space Mono
+  favicon: assets/favicon.png
+  icon:
+    repo: fontawesome/brands/github
+  name: material
+  palette:
+   # Palette toggle for dark mode
+  - scheme: default
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)